Build.ps1

```diff
@@ -4,9 +4,15 @@
   [string]$packageSuffix = "0",
   [bool]$isLocal = $false,
   [string]$outputDirectory = (Join-Path -Path $PSScriptRoot -ChildPath "buildoutput"),
-  [bool]$pack = $false
+  [bool]$pack = $false,
+  [string]$Configuration
```

Comment:
Consider setting the default in param itself:
```suggestion
  [string]$Configuration = "Release"
```

Reply:
Fixed.

Reply:
@jviau -  The variable `$Configuration` is set at runtime and might not exist, so the value passed to `Build.ps1` and `run-tests.ps1` might be an empty string. If this happens and `$Configuration` defaults to `'Release'`, the script will fail. To prevent this, check if `$Configuration` is null or empty, and if so, set it to `'Release'` within the script, which was the original approach.